sprite.cpp

```diff
@@ -9,7 +9,11 @@
 using std::get;
 
 Sprite::Sprite(const Texture *texture, const Point &home, const bool has_trail)
-	: m_texture(texture), m_home(home), m_relpos(0.0, 0.0), m_size(cfg[Cfg::SpriteSize] / 1000.0f), m_trail_start_index(0)
+	: m_texture(texture),
+	m_home(home),
+	m_relpos(0.0, 0.0),
+	m_size(get_size()),
```

Comment:
I would be careful about calling a member function in an member initializer list - it's possible that the function is called before all the members have a chance to initialize, and if the function uses an uninitialized member you're in shitsville. That's not going to happen in this case because `get_size` doesn't use any members, but we open the door to make that mistake in the future, so consider inlining the function call here or making `get_size` static.

Reply:
I did it like this to avoid having what's supposed to be the same equation being defined in two places.
Also, `get_size` is already static.

Reply:
My mistake, it's fine then